hadoop-ozone/dist/pom.xml

```diff
@@ -41,103 +41,13 @@
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-ozone-manager</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-ozone-manager.classpath
-                  </destFileName>
-                </artifactItem>
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-tools</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-tools.classpath</destFileName>
-                </artifactItem>
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-filesystem</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-filesystem.classpath</destFileName>
-                </artifactItem>
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-common</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-common.classpath</destFileName>
-                </artifactItem>
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-datanode</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-datanode.classpath</destFileName>
-                </artifactItem>
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-upgrade</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-upgrade.classpath</destFileName>
-                </artifactItem>
-                <artifactItem>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-ozone-insight</artifactId>
-                  <version>${ozone.version}</version>
-                  <classifier>classpath</classifier>
-                  <type>cp</type>
-                  <destFileName>hadoop-ozone-insight.classpath</destFileName>
-                </artifactItem>
-              </artifactItems>
+              <includes>*.classpath</includes>
```

Comment:
Fix me if I'm wrong, but `ozone classpath` now lists several modules that were previously omitted.

Previously:

```
$ ozone classpath
Usage: ozone classpath <ARTIFACTNAME>
Where the artifact name is one of:

hadoop-hdds-server-scm
hadoop-hdds-tools
hadoop-ozone-common
hadoop-ozone-csi
hadoop-ozone-datanode
hadoop-ozone-filesystem
hadoop-ozone-insight
hadoop-ozone-ozone-manager
hadoop-ozone-recon
hadoop-ozone-s3gateway
hadoop-ozone-tools
hadoop-ozone-upgrade
```

With patch:

```
$ ozone classpath
Usage: ozone classpath <ARTIFACTNAME>
Where the artifact name is one of:

hadoop-hdds-client
hadoop-hdds-common
hadoop-hdds-config
hadoop-hdds-container-service
hadoop-hdds-docs
hadoop-hdds-hadoop-dependency-client
hadoop-hdds-hadoop-dependency-server
hadoop-hdds-interface-admin
hadoop-hdds-interface-client
hadoop-hdds-interface-server
hadoop-hdds-server-framework
hadoop-hdds-server-scm
hadoop-hdds-tools
hadoop-ozone-client
hadoop-ozone-common
hadoop-ozone-csi
hadoop-ozone-datanode
hadoop-ozone-filesystem-common
hadoop-ozone-filesystem
hadoop-ozone-insight
hadoop-ozone-interface-client
hadoop-ozone-ozone-manager
hadoop-ozone-recon
hadoop-ozone-reconcodegen
hadoop-ozone-s3gateway
hadoop-ozone-tools
hadoop-ozone-upgrade
```

Invoking `ozone classpath` for the extra modules results in an error (which is the same on `master`):

```
$ ozone classpath hadoop-hdds-config
Error: Could not find or load main class org.apache.hadoop.util.Classpath
Caused by: java.lang.ClassNotFoundException: org.apache.hadoop.util.Classpath
```

Reply:
Nice catch. I did a `grep` for `OZONE_RUN_ARTIFACT_NAME` in `ozone` shell script and selected only those artifacts.